### PR TITLE
Additional Fixes for Handling Naemon and LMD Status

### PR DESCRIPTION
**Pull request by euniceremoquillo**
This commit added handling on the monitoring module for naemon service status. Commands for checking these services status were updated for a more consistent output.

This resolve MON-13199.

Signed-off-by: Eunice Remoquillo <eremoquillo@itrsgroup.com>

### CODE REVIEW

File: application/views/template.php [hunk 1]
```diff
@@ -48,16 +48,16 @@
...
 
 							// Check LMD.
-							exec('service lmd status', $output, $return_var_lmd);
+							exec('/usr/bin/systemctl status lmd.service', $output, $return_var_lmd);
 							// Check Livestatus.
-							exec('service naemon status', $output, $return_var_le);
+							exec('/usr/bin/systemctl status naemon.service', $output, $return_var_le);
 
-							if($return_var_lmd !== 0 && $return_var_le !== 0){
+							if($return_var_lmd === 127 && $return_var_le === 127){
```
Comment (jdumalaonITRS) [inline application/views/template.php:55]: Why do we need 127? From my understanding this is an exec error (command not founc) and not the systemctl return value.  refer to systemctl exit codes are in 
https://www.freedesktop.org/software/systemd/man/systemctl.html#Exit%20status

File: application/views/template.php [hunk 2]
```diff
@@ -48,16 +48,16 @@
...
 
 							// Check LMD.
-							exec('service lmd status', $output, $return_var_lmd);
+							exec('/usr/bin/systemctl status lmd.service', $output, $return_var_lmd);
 							// Check Livestatus.
-							exec('service naemon status', $output, $return_var_le);
+							exec('/usr/bin/systemctl status naemon.service', $output, $return_var_le);
 
-							if($return_var_lmd !== 0 && $return_var_le !== 0){
+							if($return_var_lmd === 3 && $return_var_le === 3){
```
Comment (jdumalaonITRS) [inline application/views/template.php:55]: Why not revert to !== 0 if the issue on error 127 is cleared up?
From the exit codes non zero seems to indicate that the service is not running

Comment (euniceremoquillo) [reply application/views/template.php:55]: to fit to the else statement, when a different scenario happens. if the services were running but might have encountered an error.

File: modules/monitoring/views/monitoring/naemon_command.php [hunk 3]
```diff
@@ -17,8 +17,13 @@
 	echo "</div>\n";
 	$this->footer = '<input style="margin-left: 12px" type="button" value="Done" onclick="history.go(-2)" />'."\n";
 } else {
-	echo '<div class="alert error" style="display: none;">'.sprintf(_('There was an error submitting your commands to %s.'), Kohana::config('config.product_name'));
-	if (!empty($result['output'])) {
+	exec('/usr/bin/systemctl status naemon.service', $output, $return_var_le);
+	if($return_var_le === 3){
```
Comment (jdumalaonITRS) [inline modules/monitoring/views/monitoring/naemon_command.php:21]: how about this one?

**Review by jdumalaonITRS — APPROVED** [review]

**Pull request merged**